consensus/hotstuff_leader.go

```diff
@@ -306,10 +315,11 @@ func (m *ConsensusModule) validatePartialSignature(msg *typesCons.HotstuffMessag
 }
 
 // TODO: This is just a placeholder at the moment for indexing hotstuff messages ONLY.
-//       It doesn't actually work because SizeOf returns the size of the map pointer,
-//       and does not recursively determine the size of all the underlying elements
-//       Add proper tests and implementation once the mempool is implemented.
-func (m *ConsensusModule) tempIndexHotstuffMessage(msg *typesCons.HotstuffMessage) {
+//
+//	It doesn't actually work because SizeOf returns the size of the map pointer,
```

Comment:
Non-Blocker: 

Revert, Goland doesn't recognize this as part of the TODO anymore... Maybe that was the intention

Reply:
Done, VSCode formats files on save automatically, I need to configure it to mimic Goland I suppose. I'll look into it